README.md

```diff
@@ -191,6 +191,62 @@ This option can also be set using `-alertmanager.uris` flag. Example:
 
 This variable is required and there is no default value.
 
+#### ANNOTATIONS_DEFAULT_HIDDEN
```

Comment:
Wouldn't it make more sense for these to be user-specific options configured in the UI?